Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -104,6 +107,9 @@ extension AssetsCatalogParser {
     let type = item.extension ?? ""
 
     switch (type, AssetCatalog.Item.supported.contains(type)) {
```

Comment:
According to the docs (https://developer.apple.com/library/content/documentation/Xcode/Reference/xcode_ref-Asset_Catalog_Format/AssetTypes.html), the group type has no extension, and no periods are allowed in the name of the group. 

Following that documentation, option 2 is actually the correct one, and we should ignore groups with periods in them. If, by some quirk in Xcode, periods are allowed, I don't think we should support it unless the documentation changes.